docs/resources/ui-building-blocks/components/built-in-components/pincode.md

```diff
@@ -0,0 +1,339 @@
+---
+slug: pincode
+title: PinCode
+tags: []
+description: Learn how to add PinCodewidget in your FlutterFlow app.
+sidebar_position: 4
+---
+# PinCode
+
+The PinCode widget allows you to enter the PIN or OTP. You could use this widget to verify the user identity or a transaction before making payments in fintech apps.
+
+## Adding PinCode widget
+
+To add a PinCode widget:
+
+1. Open the [Widget Palette](../../../../../intro/ff-ui/widget-palette) and locate the **PinCode** widget under the **Base Elements** tab. You can drag it into your desired location or add it directly from the widget tree or canvas area.
```

Comment:
Link doesnt go to the widget palette page.

Reply:
corrected!